### PR TITLE
Report extended counters for Metered and Histogram

### PR DESCRIPTION
**Pull request by dkaukov**

### CODE REVIEW

**Comment by mikebell90** [comment]
Example: Why would I want to send a DISABLED metric. Yet you've inverted the logic to only send COUNT when it's disabled.

**Comment by mikebell90** [comment]
Also every fork from the original deserves at least one, possibly two lines of comments? Just how do you think we'll keep this in sync without breaking?

**Comment by mikebell90** [comment]
oh and add a changelog

File: otj-metrics-core/src/main/java/com/opentable/metrics/graphite/OtGraphiteReporter.java [hunk 1]
```diff
@@ -365,8 +367,11 @@ private void reportTimer(String name, Timer timer, long timestamp) throws IOExce
         reportMetered(name, timer, timestamp);
     }
 
+    /**
+     * CHANGE: call {@link OtGraphiteReporter#reportCounter(String, Counter, long)}
```
Comment (adamzr) [inline otj-metrics-core/src/main/java/com/opentable/metrics/graphite/OtGraphiteReporter.java]: What is this JavaDoc trying to say?

Comment (dkaukov) [reply otj-metrics-core/src/main/java/com/opentable/metrics/graphite/OtGraphiteReporter.java]: That we inserted call to the reportCounter() ;)

Comment (dkaukov) [reply otj-metrics-core/src/main/java/com/opentable/metrics/graphite/OtGraphiteReporter.java]: It is attempt to document changes to the original class

File: otj-metrics-core/src/main/java/com/opentable/metrics/graphite/OtGraphiteReporter.java [hunk 2]
```diff
@@ -366,7 +366,9 @@ private void reportTimer(String name, Timer timer, long timestamp) throws IOExce
     }
 
     private void reportMetered(String name, Metered meter, long timestamp) throws IOException {
-        sendIfEnabled(COUNT, name, meter.getCount(), timestamp);
+        if (getDisabledMetricAttributes().contains(COUNT)) {
```
[the remark this answers is not in the document]
Comment (dkaukov) [reply otj-metrics-core/src/main/java/com/opentable/metrics/graphite/OtGraphiteReporter.java]: I've changed that

File: otj-metrics-core/src/main/java/com/opentable/metrics/graphite/OtGraphiteReporter.java (hunk 1 above)
Comment (adamzr) [reply otj-metrics-core/src/main/java/com/opentable/metrics/graphite/OtGraphiteReporter.java]: It sounds to me like a TODO, i.e. this should be changed. Can you rephrase it?

Comment (dkaukov) [reply otj-metrics-core/src/main/java/com/opentable/metrics/graphite/OtGraphiteReporter.java]: Done

**Comment by mikebell90** [comment]
Looks cleaner to me @adamzr what do you think?
